packages/cli/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@redocly/cli",
-  "version": "1.0.0-beta.106",
+  "version": "1.0.0-beta.107",
```

Comment:
Please also update the version here: https://github.com/Redocly/redocly-cli/pull/807/files#diff-6e2e2a1851648938b325ba84de634407a4e69a644ea61102df15ca4a8a7a9758R38